src/snapshot.ts

```diff
@@ -12,7 +12,7 @@ Core functionality for creating a snapshot of a project's dependencies.
 
 /**
  * When multiple snapshots are submit, Job provides the means for Snapshots to
```

Comment:
I know this isn't part of this diff, but
```suggestion
 * When multiple snapshots are submitted, Job provides the means for Snapshots to
```